modules/ROOT/pages/test-with-munit-in-studio.adoc

```diff
@@ -0,0 +1,96 @@
+
+== Get to Know MUnit Tests
+
+You can perform the following tasks with MUnit in Studio:
+
+//TODO: Make the list of tasks you can perform in Studio with MUnit and create a link from each bullet to the content on this very same page
+* Create a test
+* Create a test suite
+* Run a test
+* Debug your project
+* Fix errors
+* Generate test reports
+
+When you work with your MUnit tests in Studio, you can:
+
+* Find the test of your project in the `src/test/munit` folder, in *Package Explorer*.
+* Drag components to your test from the *Mule Palette*, in *Message Flow*.
+* Define the MUnit configuration for your tests from *Global Elements*.
+* Review test XML files from *Configuration XML*.
+
+== Create MUnit Tests and MUnit Test Suits
+
+To create an MUnit test:
+
+. Open your Mule project in Studio.
+. Right-click on the *src/test/munit* folder, or create this folder if it doesn't exist in your Mule project.
+. Select *New* > *MUnit Test*.
+. In the *MUnit test suite* window, write a name for your test suite.
+. Select the flows to test from the list of flows available in your project.
+. Click *Finish*.
+
+[NOTE]
+You can also configure MUnit tests and test suites from the XML editor. 
+
+== Run MUnit Tests
+
+You can run MUnit test in different ways in Studio:
+
+* Right-click on your MUnit test suite file and select *MUnit* > *Run Test*.
+* In *Message Flow*, right-click on your test flow, and select *Run MUnit suite*.
+
+When you run a test, Studio creates a build and displays the progress and the result of the test in the *Console* tab and the *MUnit* window. If all tests execute successfully, the build succeeds. If there is an error, the build fails. 
+
+// == Monitor Your Tests and Review the Test Results
+// If the test results indicate any failures, you can learn more about these failures in the error section.
+//TODO: Create a troubleshooting section for the user to learn about possible errors, if any.
+
+== Run MUnit Tests in Debug Mode
+
+You can run your MUnit tests in debug mode to step through your code, inspect variables, and understand the flow of your Mule application.
+
+. Open your Mule project in Studio.
+. In the *src/test/munit* folder, open the MUnit test suite to debug.
+. In *Message Flow*, add a breakpoint by right-clicking on the element that must have the breakpoint. 
+. Right-click on your MUnit flow test and select *Debug MUnit test*.
+
+== Enable Flow Sources
+
+The `Enable Flow Sources` element within the MUnit test tells MUnit which flow sources to load and execute for the test.
+By default, MUnit doesn't start the event sources (flow sources such as triggers and listeners). 
+
+You can define flow sources with the *Mule Properties* view of your MUnit test in Studio:
+
+. Click *Add*.
+. Select the name of the flow from *Name of the flow*.
+
+Learn more about flow sources in MUnit docs (xref:munit::enable-flow-sources-concept.adoc[]) and configure which flow sources must start for your tests.
```

Comment:
We have so many names for these (event sources, triggers, flow sources, and more). Let's deal with that later.

Reply:
Sure! Thanks!